toolkit/tools/internal/rpm/rpm.go

```diff
@@ -536,7 +536,7 @@ func buildAllSpecsList(baseDir string) (specPaths []string, err error) {
 
 	specPaths, err = filepath.Glob(specFilesGlob)
 	if err != nil {
-		logger.Log.Errorf("Failed while trying to enumerate all spec files with (%s). Error: %v.", specFilesGlob, err)
+		return nil, fmt.Errorf("failed to enumerate all spec files with (%s)\n%w", specFilesGlob, err)
```

Comment:
nit: For functions that use named output values, it is common to write the following:

```golang
err = nil, fmt.Errorf("failed to enumerate all spec files with (%s):\n%w", specFilesGlob, err)
return
```

Though, I am fine with either.

Reply:
I think you meant 
`specPaths, err = nil, fmt.Errorf("failed to enumerate all spec files with (%s):\n%w", specFilesGlob, err)`
`return`
Changed it. It reads better now.

Reply:
If an error is returned, it doesn't really matter what the value of `specPaths` is.